AppController/test/tc_djinn.rb

```diff
@@ -1329,4 +1329,142 @@ def test_set_deployment_id
   end
 
 
+  def get_djinn_mock
+    role = {
+        "public_ip" => "app_public_ip1",
```

Comment:
Since these values are not really used anywhere, please simplify to something like:

"public ip" => "my public ip"
"private ip" => "my private ip"

I believe they are not related to an app anyway.